src/utils/pools.tsx

```diff
@@ -1360,16 +1169,44 @@ async function _addLiquidityNewPool(
 
   // All instructions didn't fit in single transaction
   // initialize and provide inital liquidity to swap in 2nd (this prevents loss of funds)
-  tx = await sendTransaction(
-    connection,
-    wallet,
-    instructions.concat(cleanupInstructions),
-    [tokenSwapAccount, ...signers]
-  );
+  tx = await sendTransaction(connection, wallet, instructions.concat(cleanupInstructions), [
+    tokenSwapAccount,
+    ...signers,
+  ]);
 
   notify({
-    message: "Pool Funded. Happy trading.",
-    type: "success",
+    message: 'Pool Funded. Happy trading.',
+    type: 'success',
+    description: `Transaction - ${tx}`,
+  });
+}
+
+export async function setFreezeAuthorityBitMask(
```

Comment:
This is a new action that isn't hooked up anywhere in the UI but I used it with a transient button to successfully integration test the functionality in the contract. Can be hooked up to a UI later.